palantir-java-format-native/src/main/resources/META-INF/native-image/reachability-metadata.json

```diff
@@ -164,26 +164,19 @@
       "glob": "META-INF/services/java.lang.System$LoggerFinder"
     },
     {
-      "glob": "com/sun/tools/javac/resources/compiler_en.properties"
-    },
-    {
-      "glob": "com/sun/tools/javac/resources/compiler_en_US.properties"
+      "glob": "com/sun/tools/javac/resources/**"
     },
     {
       "module": "jdk.compiler",
-      "glob": "com/sun/tools/javac/resources/compiler_en.properties"
-    },
-    {
-      "module": "jdk.compiler",
-      "glob": "com/sun/tools/javac/resources/compiler_en_US.properties"
+      "glob": "com/sun/tools/javac/resources/**"
     }
   ],
   "bundles": [
     {
-      "name": "com.sun.tools.javac.resources.compiler",
```

Comment:
we shouldn't restrict the locales for the compiler bundle - by default we will include everything